templates/zerver/help/configure-authentication-methods.md

```diff
@@ -0,0 +1,27 @@
+# Configure authentication methods
+
+{!admin-only.md!}
+
+{!go-to-the.md!} [Authentication methods](/#organization/auth-methods)
+{!admin.md!}
+
+2. Enable or disable the authentication methods you want.
+
+     * **Email** - users can create an account using their Email to log in.
+
+     * **GitHub** - users can use their GitHub account to log in.
+
+     * **Google** - users can use their Google account to log in.
+
+     * **LDAP** - users can use an LDAP username and password to authenticate.
+
+     * **RemoteUser** - a Single-Sign-On system can be used to log in.
+
+     * **Dev** - only for development. Allows passwordless login as any user.
+
```

Comment:
For all of the options, can you fix the captiatlization (users instead of Users) and use more active verbs instead of passive voice in your descriptions?